src/ORM/Association/BelongsTo.php

```diff
@@ -33,6 +33,13 @@ class BelongsTo extends Association
     use SelectableAssociationTrait;
 
     /**
+     * Valid strategies for this type of association
+     *
+     * @var array
+     */
+    protected $_validStrategies = [parent::STRATEGY_JOIN];
```

Comment:
You could use a select. This can be useful when the association is in a different database/connection.